src/main/java/edu/ie3/datamodel/io/factory/typeinput/SystemParticipantTypeInputFactory.java

```diff
@@ -50,7 +50,6 @@
 
   // StorageTypeInput
   private static final String P_MAX = "pmax";
-  private static final String CP_RATE = "cprate";
```

Comment:
Are these changes addressing already all changes we discussed with @sensarmad? And if yes, are they comprehensive? Or whats the reason for changing them here?

Reply:
I discussed them all with @sensarmad. It was on the one hand to harmonize naming with the biomass model, which also has a maximum rate of change of power and on the other hand, bringing a more comfortable unit into play for the life time. The initial changes, you have been involved in discussion have already been addressed in a different PR. Do you feel there is something missing?